src/Symfony/Component/Messenger/Command/FailedMessagesRetryCommand.php

```diff
@@ -154,7 +154,7 @@ private function runInteractive(SymfonyStyle $io, bool $shouldForce)
         }
```

Comment:
Why it is only collecting one message at a time a few lines above `foreach ($receiver->all(1) as $envelope) {`? Wouldn't it be better to batch more messages to avoid querying doctrine for each message?

Reply:
Hmm. We could... but I don't see a lot of benefit. This is the interactive mode, and we're asking the user if they want to retry one-by-one. In theory, something else could also be reading from the failure transport at the same time, so getting whatever the latest id is one-by-one seems sensible.